uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/App.java

```diff
@@ -108,6 +127,17 @@ public void launch() {
         CommonUtils.launchApp(packageName);
     }
 
+    @Override
+    public void update() {
+        if (updateAppInstaller instanceof LocalApkInstaller && !TextUtils.isEmpty(localUpdateApkFileName)) {
+            Logger.i(TAG, "Updating the " + this.appName + " from local apk..");
+            AdbShellUtils.updatePackage(LocalApkInstaller.LOCAL_APK_PATH_PREFIX + localApkFileName, "-t", "-r", "-d");
```

Comment:
Why not put this as a method on the `LocalApkInstaller`? I mean create a new method in that file called as `updateApp` - similar to what you've done for PlayStore, and then over here you can probably just invoke `updateAppInstaller.updateApp(identifier)`

Reply:
addressed this in f9ecbb25a67397feee33a63807075d46e1b84038